Pod/Classes/MRGMarqueeLabel.m

```diff
@@ -95,43 +85,66 @@ - (CGSize)intrinsicContentSize {
 
 - (void)layoutSubviews {
     [super layoutSubviews];
-    [self layoutLabels];
+    
+    if (self.needsAnimationReset || !CGRectEqualToRect(self.previousRect, self.bounds)) {
```

Comment:
You could have compared `self.bounds` to `self.contentView.frame` instead of adding a private `CGRect` variable?